concent_api/command_line_tool/concent_request.py

```diff
@@ -0,0 +1,102 @@
+import os
+import sys
+import argparse
+import json
+
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+from message_handler import MessageHandler
+from message_extractor import MessageExtractor
+from key_manager import KeyManager
+
+
+def get_json_data(message_file, message_str):
```

Comment:
I see this is something that comes from client (who is command line user in this case), so please add checking if file exists and handle situation when it does not.

Reply:
I think there should be some kind of error/warning if both values are passed.

Reply:
When both values are passed, error is handled automatically by argparse e.g. error: argument -s/--message: not allowed with argument -f/--message-file

Reply:
Error handling will be added in next iteration.